spec/features/admin_views_all_orders_spec.rb

```diff
@@ -0,0 +1,154 @@
+
+    visit "/login"
+    fill_in "Username", with: "admin"
+    fill_in "Password", with: "password"
+    click_on "Login to your account"
+  end
+
+  scenario "admin sees all orders and statuses" do
+    visit admin_dashboard_index_path
+    within ".orders" do
+      expect(page).to have_content "##{@order1.id}"
+      expect(page).to have_content "ordered"
+      expect(page).to have_content "March 1, 2016"
+
+      expect(page).to have_content "##{@order2.id}"
+      expect(page).to have_content "cancelled"
+      expect(page).to have_content "February 13, 2016"
+
+      expect(page).to have_content("Ordered: 1")
+      expect(page).to have_content("Paid: 0")
+      expect(page).to have_content("Cancelled: 1")
+      expect(page).to have_content("Completed: 0")
+    end
+  end
+
+  scenario "admin filters by status" do
+    visit admin_dashboard_index_path
+    click_on "Ordered"
+
+    within ".orders" do
+      expect(page).to have_content "##{@order1.id}"
+      expect(page).to have_content "ordered"
+      expect(page).to have_content "March 1, 2016"
+
+      expect(page).to_not have_content "##{@order2.id}"
+      expect(page).to_not have_content "cancelled"
+      expect(page).to_not have_content "February 13, 2016"
+    end
+
+    visit admin_dashboard_index_path
+    click_on "Paid"
+
+    within ".orders" do
+      expect(page).to_not have_content "##{@order2.id}"
+      expect(page).to_not have_content "cancelled"
+      expect(page).to_not have_content "February 13, 2016"
+    end
+
+    visit admin_dashboard_index_path
+    click_on "Cancelled"
+
+    within ".orders" do
+      expect(page).to have_content "##{@order2.id}"
+      expect(page).to have_content "cancelled"
+      expect(page).to have_content "February 13, 2016"
+
+      expect(page).to_not have_content "##{@order1.id}"
+      expect(page).to_not have_content "ordered"
+      expect(page).to_not have_content "March 1, 2016"
+    end
+    visit admin_dashboard_index_path
+    click_on "Completed"
+
+    within ".orders" do
+      expect(page).to_not have_content "##{@order2.id}"
+      expect(page).to_not have_content "cancelled"
+      expect(page).to_not have_content "February 13, 2016"
+    end
+  end
+
+  scenario "admin tries to visit status path that does not exist" do
+    visit "/admin/notathing"
+
+    expect(page).to have_content "Status is invalid"
+    within ".orders" do
+      expect(page).to have_content "##{@order1.id}"
+      expect(page).to have_content "ordered"
+      expect(page).to have_content "March 1, 2016"
+
+      expect(page).to have_content "##{@order2.id}"
+      expect(page).to have_content "cancelled"
+      expect(page).to have_content "February 13, 2016"
+
+      expect(page).to have_content("Ordered: 1")
+      expect(page).to have_content("Paid: 0")
+      expect(page).to have_content("Cancelled: 1")
+      expect(page).to have_content("Completed: 0")
+    end
+  end
+
+  context "admin tries to change status of a single order" do
```

Comment:
To be more clear, this should be something like:
  scenario "sees notification of status update" do